### PR TITLE
fix: navigation home item color for light theme

### PR DESCRIPTION
**Pull request by darmiel**
**What this PR does / why we need it**:

When changing the theme from dark to light, the text color of the "Home" item changes from white to black which impairs readability. This PR adds the color of the navbar title (white) to the home item.

<table>
  <tr>
    <th>dark</th>
    <th>light</th>
    <th>light (expected)</th>
  </tr>
  <tr>
    <td>
      Screenshot 2023-01-13 at 20 02 58
    </td>
    <td>
      Screenshot 2023-01-13 at 20 03 12
    </td>
    <td>
      Screenshot 2023-01-13 at 20 07 35
    </td>
  </tr>
<table>

**Which issue(s) this PR fixes**:
-

**Special notes for your reviewer**:
-

**Release note**:

```bugfix user
NONE
```

### CODE REVIEW

**Comment by gardener-robot-ci-1** [comment]
Thank you @darmiel for your contribution. Before I can start building your PR, a member of the organization must set the required label(s) {'reviewed/ok-to-test'}. Once started, you can check the build status in the PR checks section below.

File: frontend/src/components/MainNavigation.vue [hunk 1]
```diff
@@ -132,7 +132,7 @@ SPDX-License-Identifier: Apache-2.0
             <v-icon>mdi-home-outline</v-icon>
```
Comment (grolu) [inline frontend/src/components/MainNavigation.vue:132]:
```suggestion
            <v-icon small color="main-navigation-title">mdi-home-outline</v-icon>
```

**Comment by grolu** [comment]
Please also fix the color for the Home Icon

**Review by grolu — APPROVED** [review]
/lgtm

**Pull request merged**